docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,292 @@
+Remote Attestation Collaterals
+=====
+
+Adding attestation APIs to support collaterals for remote attestation.  Collaterals are information from a second source that the user/challenger can provide for attestation verification.
+
+Motivation
+----------
+
+Currently, the existing collaterals used for Intel SGX quote verification are not exposed to the user.  Adding these new APIs allows the user to specify these collaterals instead of using the default collaterals used by OE SDK.  This enables two possible user scenarios:
+1. The user/challenger has the option to specify its set of collaterals during verification.
```

Comment:
Maybe we could describe in a way to convey the following idea (not exact words:

The existing SDK does not allow an enclave client to provide any policy input during the validation process. The policy was fixed in OE SDK's choosing. 

We are giving users the control of the quote validation policy by providing new APIs to provide collaterals (validation policy). They can control the policy (collaterals) used in the validation of their quote as long as they provide the collaterals in the expected format.

Reply:
Updated.